src/main/java/org/wso2/carbon/mongodb/query/MongoPreparedStatementImpl.java

```diff
@@ -491,6 +547,38 @@ private void getUpdateObject(JSONObject object) {
         this.query = new BasicDBObject(mapQuery);
     }
 
+    /**
+     * get the unset object.
+     *
+     * @param object to update
+     */
+    private void getUnsetObject(JSONObject object) {
```

Comment:
Returning void on getter method looks odd. There are some other similar wrong pattern on the existing code in this class. Yet new code need to be done properly.